README.md

```diff
@@ -1,58 +1,24 @@
 # jamsocket-jupyter-notebook
-Scaffold for setting up a hosted jupyter notebook on the Jamsocket platform
+Base repo for running Jupyter Notebooks on the Jamsocket platform
 
-## To run jupyter-notebook on Jamsocket via a spawn webpage:
+## Steps
 
-```bash
-# first, make sure you're logged in to the Jamsocket CLI (send us an email if you need credentials!)
-npx jamsocket login
-
-# create a service called jupyter-notebook
-npx jamsocket service create jupyter-notebook
-
-# build the jupyter notebook docker image
-docker build . -t jupyter-notebook                        # if you're building on an Intel/AMD machine
-docker build . -t jupyter-notebook --platform=linux/amd64 # if you're building on Apple Silicon or ARM-based machine
-
-# push the image to the jupyter-notebook service
-npx jamsocket push jupyter-notebook jupyter-notebook
-```
-
-(Notice: if you are building on an ARM-based processor - like Apple Silicon - you may need to add `--platform=linux/amd64` to the docker build command.)
-
-After building and pushing the jupyter-notebook image to jamsocket, generate a spawn token using the CLI.
-
-```bash
-npx jamsocket token create jupyter-notebook
-```
-
-Take the returned token and replace `[SPAWN TOKEN HERE]` in `spawn.html`.
-
-Try it out locally by opening `spawn.html` in your browser:
-
-```bash
-open spawn.html
-```
-
-## Jamsocket Guide
-
-For more information about the Jamsocket platform, check out our [docs](https://docs.jamsocket.com/).
+1. Fork this repository.
+2. Add your notebook(s) to the `/notebook` directory.
+3. Add your notebook's python dependencies to `requirements.txt`.
+4. Update the `NOTEBOOK` value in `run.sh` to make sure Jupyter opens your notebook on start up. 
+5. Add `JAMSOCKET_USER` and `JAMSOCKET_TOKEN` values to your GitHub secrets. (Contact us to get Jamsocket credentials.)
+6. Push your repo to your GitHub account, and your Jupyter Notebook container will be automatically built and pushed to Jamsocket.
+7. Get a Spawn URL from us.
```

Comment:
“When users land on this URL, they'll be directed to a newly-spawned Jupyter notebook” or something?